src/main/kotlin/stringcalculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,27 @@
+package stringcalculator
+
+import java.lang.IllegalArgumentException
+
+class StringAddCalculator {
+
+    fun add(text: String): Int {
+        if (text.isBlank()) return 0
+        if (text.isEmpty()) return 0
+        if (text.length == 1) return text.toInt()
+        val splittext = text.split(",", ":")
+        val result = Regex("//(.)\n(.*)").find(text)
+        result?.let {
+            val customDelimiter = it.groupValues[1]
+            val tokens = it.groupValues[2].split(customDelimiter)
+            tokens.forEach{if (it.toInt() < 0) throw RuntimeException("$it is negative") }
```

Comment:
여기서 사용하는 it은 result?.let의 it인지 forEach의 it인지 구분이 힘들지 않을까요?

이렇게 중첩되는 경우 두번째나 첫번째에 인자를 명시해주면 구분이 좀 쉬워질 것 같습니다.

```suggestion
            tokens.forEach { token -> if (token.toInt() < 0) throw RuntimeException("$token is negative") }
```

Reply:
요청 하신대로 구분해 보았습니다.